### DISCUSSION TITLE
Suggestion: Add Multiplayer

### DISCUSSION THREAD

**Discussion by Jerunnon** [voxeltycoon/issues#379] [General]
I really like your game it is awesome. If I could wish for something than it is multiplayer. With Multiplayer I don't meen somethin like in SimCity, No. I would love a multiplayer were you can work together on one map and one city. Or at least were you would need to work together in order to manage all the difficulties and tasks in the game.

**Comment by ndelta0** [comment]
It's in roadmap planned for later date
